INSTALL

```diff
@@ -325,7 +325,32 @@ can get the libpng and freetype requirements (darwinports, fink,
 /usr/X11R6) and the different architectures (e.g., x86, ppc, universal) and
 the different OSX version (e.g., 10.4 and 10.5). We recommend that you build
 the way we do for the OSX release: get the source from the tarball or the
-git repository and follow the instruction in :file:`README.osx`.
+git repository and install the required dependencies through a third-party
+package manager: two widely used package managers are homebrew, and MacPorts.
```

Comment:
Suggest the was clarify that this whole section is for is when you're *not* using conda

Also suggest:
"through a third-party package manager such as `homebrew <https://brew.sh/>`_ or `MacPorts <https://www.macports.org/>`_"